apps/web/src/server/applications/lib/services/case.service.js

```diff
@@ -101,3 +102,40 @@ export const updateCase = async (caseId, payload) => {
 export const getCaseDocumentationCategories = (id) => {
 	return get(`applications/${id}/documents`);
 };
+
+/**
+ * Get the parent folder(s) for the current folder - used for breadcrumbs etc.
+ *
+ * @param {number} caseId
+ * @param {DocumentationCategory |undefined} documentationCategory
+ * @returns {DocumentationCategory[]}
+ */
+export const getCaseDocumentationFolderTree = (caseId, documentationCategory) => {
+	// TODO:Mock Version:
+	/** @type {DocumentationCategory[] } */
+	let folderTree = [];
+
+	if (caseId && documentationCategory) {
+		folderTree = [documentationCategory];
```

Comment:
I understand in reality you won't have this object as it comes from the API, right?
If so, can you define documentationCategory directly in this function and remove it from the controller?

Reply:
I am assuming that it will need to know the details of the current folder we are showing, in order to get the folder tree for breadcrumbs.  The details are passed in in documentCategory.